pkg/common/cns-lib/volume/util.go

```diff
@@ -374,8 +394,12 @@ func ExtractFaultTypeFromVolumeResponseResult(ctx context.Context,
 			log.Infof("Extract vimfault type: %+v  vimFault: %+v Fault: %+v from resp: %+v",
 				faultType, fault.Fault, fault, resp)
 			slice := strings.Split(faultType, ".")
-			vimFaultType := vimFaultPrefix + slice[1]
-			return vimFaultType
+			if classifyAsNonStorageFault(slice[1]) {
+				faultType = csifault.CSINonStorageFaultPrefix + slice[1]
```

Comment:
This changes line#381. Does it affect the callers of this function?

Reply:
Updated method doc. No it doesn't affect the callers of this function.